.github/workflows/publish-sample.yml

```diff
@@ -0,0 +1,40 @@
+name: publish-sample
+
+on: 
+ workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+
+permissions:
+  contents: read
+
+jobs:
+  build-publish-release:
+    name: "release"
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    permissions:
+      contents: write
+      packages: write
+    steps:
+       
+      # need to convert repo name to lowercase for Docker
+      - name: Get repo
+        run: |
+          echo "REPO_OWNER=$(echo $GITHUB_REPOSITORY_OWNER | awk '{print tolower($0)}')" >> $GITHUB_ENV
```

Comment:
In our publishpackage workflow which publishes the images to ghcr, we use the `github.repository` to get the `deislabs/ratify` portion of the image ref. Will this be the equivalent? https://github.com/deislabs/ratify/blob/8f65ad2bf95f5baba6fd454bdd817cf24e0927a6/.github/workflows/publish-package.yml#L29

Reply:
relevant docs it looks like: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
thanks for the tip!, ive updated the env various, and here what the output looks like on my fork:

Copied [registry] wabbitnetworks.azurecr.io/test/notary-image:signed => [registry] ghcr.io/susanshi/ratify/notary-image:signed
Digest: sha256:[8](https://github.com/susanshi/ratify/actions/runs/5384401919/jobs/9772285530#step:4:9)e3d01113285a0e4aa574da8eb9c0f112a1eb979d72f73399d7175ba3cdb1c1b